llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -3477,8 +3474,8 @@ class VPlan {
   /// middle VPBasicBlock. If a check is needed to guard executing the scalar
   /// epilogue loop, it will be added to the middle block, together with
   /// VPBasicBlocks for the scalar preheader and exit blocks.
```

Comment:
nit: would be good to explain what `IdxTy` stands for, and/or use a more descriptive name.